homeassistant/util/package.py

```diff
@@ -103,11 +109,24 @@ def install_package(
         args.append("--upgrade")
     if constraints is not None:
         args += ["--constraint", constraints]
+    not_venv = not is_virtual_env()
     if target:
-        assert not is_virtual_env()
+        assert not_venv
         # This only works if not running in venv
-        args += ["--user"]
-        env["PYTHONUSERBASE"] = os.path.abspath(target)
+        abs_target = os.path.abspath(target)
+        env["PYTHONUSERBASE"] = abs_target
```

Comment:
Do we still need it? I missed this section when working on https://github.com/home-assistant/core/pull/125110